src/performance/optimizer.py

```
@@ -0,0 +1,155 @@
+        for suggestion in suggestions:
+            if suggestion["severity"] == "high":
+                # Apply immediate optimizations
+                optimization = self._apply_optimization(operation, suggestion)
+                if optimization:
+                    optimizations.append(optimization)
+                    
+        if optimizations:
+            self.optimizations[operation].extend(optimizations)
+            self.optimization_history.append({
+                "operation": operation,
+                "timestamp": datetime.now(),
```

Ruff (0.8.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)